test/unit/nuke_util/test_ocio.py

```diff
@@ -109,6 +109,7 @@ def test_create_config_from_file(
     create_from_file.assert_called_once_with(custom_ocio_config_path)
 
 
+@pytest.mark.skipif(os.name == "nt", reason="Does not run on Windows")
```

Comment:
I see this is from the previous PR, but I'm wondering what error we're seeing because I'd like to avoid commenting out a test on an OS without understanding why.

My hunch is that the mock for the OCIO config isn't set-up correctly for windows:

https://github.com/aws-deadline/deadline-cloud-for-nuke/blob/35b9ceb28598dacfa7337e717681f91e01e57228/test/unit/nuke_util/test_ocio.py#L14-L18

and that we're failing on this check because of that:

https://github.com/diane-hj/deadline-cloud-for-nuke/blob/35b9ceb28598dacfa7337e717681f91e01e57228/src/deadline/nuke_util/ocio.py#L30-L32

`PurePath` will evaluate to the current OS flavour, these absolute posix paths don't look absolute on windows. Here's a Local run from my machine:
```
>>> PurePath("/asdf/asdf")
PurePosixPath('/asdf/asdf')
>>> PurePath("/asdf/asdf").is_absolute()
True
>>> PureWindowsPath("/asdf/asdf").is_absolute()
False
```

So the only thing we should need to do here to get this test to run is to conditionally set the mock for it's OS. Example:

```python
if os.name == "nt":
    return MockOCIOConfig(
        working_dir="C:/this/ocio_configs", search_paths=["luts", "C:/my/absolute/luts"]
    )
return MockOCIOConfig(
        working_dir="/this/ocio_configs", search_paths=["luts", "/my/absolute/luts"]
    )
```

Reply:
Thanks Morgan! As discussed offline, I created a separate Github issue for this: https://github.com/aws-deadline/deadline-cloud-for-nuke/issues/164